### ISSUE TITLE
Bug: glTFv2: FB_ngon_encoding is always set, even if not used

### ISSUE THREAD

**Issue by RichardTea** [assimp/assimp#4618]
**Describe the bug**
The glTF v2 exporter always adds `FB_ngon_encoding` to `extensionsUsed`, even when it is not used. 

**To Reproduce**
1. Create an assimp scene where all meshes are entirely triangles
2. Export this scene to glTFv2
3. Look in `extensionsUsed` in the exported glTF file
4. Note that `FB_ngon_encoding` is present in `extensionsUsed`, even though no primitives actually used it.

**Expected behavior**
FB_ngon_encoding should _only_ be set if it was used - one or more meshes were triangulated during the glTF export.

**Additional context**
Applications that understand the (draft) FB_ngon_encoding extension will enable their support on finding that extension, and thus loading the scene will take longer than it should.

There are also some uses of glTFv2 that only support 'core' glTF files that do not use any extensions at all, even optional ones, and warn unnecessarily when loading glTF exported by assimp.

**Comment by kimkulling** [comment]
Right, this flag is always true. The export is entirely independent of the import. The export is based on the aiScene instance. And, indeed, this class is aware of this encoding based on our post-processing so that the export can contain these encoded primitives.

So from my point of view setting the flag to true is correct.

**Comment by RichardTea** [comment]
@kimkulling Sorry, you've misunderstood.

`FB_ngon_encoding` tells an application loading the glTF "This glTF contains meshes that have quads and ngons"

When I use assimp to export a pair of triangles then load the result in an application that does understand `FB_ngon_encoding`, I will sometimes get two triangles (correct) and sometimes get a quad (wrong!)

`FB_ngon_encoding` should only be set if the aiScene assimp exported did in fact contain quads or ngons.

**Comment by RichardTea** [comment]
The upstream FB_ngon_encoding extension seems to have been abandoned for the time being, so I'd suggest removing it.

**Comment by tellypresence** [comment]
glTF import issues tracked via epic #6162
